package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Service",
   "main": "index.js",
   "scripts": {
+    "dev": "./start_db.sh && npx nodemon index.ts",
```

Comment:
Please install nodemon as dev dependency and use this script without npx.
If your `start_db.sh` fails for whatever reason (example you don't have internet connection) you will start the app without database right?
We don't want this.

`dev` script should only start nodejs server and it shoud be fast.
`start_db.sh` can be executed once and you have a running container until you stop it.  There is no reason to delete container, start it again and initialize database on every app start.

Reply:
`So, npm run dev should only start the app, and we should use start_db.sh to start the database container?` - Yes, just like that. 

And since you are using `npx` in the `start_db.sh` you will always need internet connection :)

Reply:
`nodemon is already installed as dev dependency. We tried this script without npx part, but it didn't work.`
That's weird, we will fix that.